config/crd/bases/redhatgov.io_giteas.yaml

```diff
@@ -47,6 +47,13 @@ spec:
                 description: Defines the desired state of the Gitea Deployment
                 type: object
                 properties:
+                  trust_bundle_configmap:
```

Comment:
Can we change this to `trustBundleConfigmap`? That allows us to keep with kubernetes object convention of using camel case for attribute names. The Ansible operator ask will convert this back to snake case before it's passed in as an Ansible fact 😄

Reply:
Yep, I sure can. Let me fix :)